src/ppx/grfx/vk/vk_device.cpp

```diff
@@ -499,6 +499,7 @@ Result Device::CreateQueues(const grfx::DeviceCreateInfo* pCreateInfo)
         uint32_t queueFamilyIndex = ToApi(pCreateInfo->pGpu)->GetGraphicsQueueFamilyIndex();
         for (uint32_t queueIndex = 0; queueIndex < pCreateInfo->graphicsQueueCount; ++queueIndex) {
             grfx::internal::QueueCreateInfo queueCreateInfo = {};
+            queueCreateInfo.commandType                     = grfx::COMMAND_TYPE_GRAPHICS;
```

Comment:
Do you know if this all works fine if we have a single queue family supporting graphics, compute and transfer? Do they all point to the same device queue?

Reply:
Tried this on my desktop GPU (which has 1 graphics|compute queue + 1 compute queue), and with llvmpipe (a single graphics|compute queue).

On llvmpipe, the selected queue is the only one (makes sense), and for my real GPU, one queue for compute, and one for graphics. Transfer is shaders with graphics.

Reply:
makes sense, thank you!